go.mod

```diff
@@ -0,0 +1,163 @@
+module github.com/meshery/meshery-perf
```

Comment:
We would not need this file. As far as I know, meshery perf will not register any definitions and schemas. So for that reason, this file is not needed. We are not registering any components here